flux_led/__main__.py

```diff
@@ -877,14 +891,19 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
         #
         # For all other rgb and rgbw devices, the value is 00
 
-
-
-        # Some devices provide RGBW control, but require two separate writes
-        # If we've been given both colours and whites, split them up
-        if self.badrgbw == True and (r != None and (w != None or w2 != None)):
-            self.setRgbw(w=w, w2=w2, retry=retry, persist=persist)
-            self.setRgbw(r, g, b, retry=retry, persist=persist)
-            return
```

Comment:
The app only sends independent requests, that's correct. The app doesn't have a screen where both whites and colors can be set at the same time.

However if I use the flux_led command-line, I can set all 5 channels simultaneously. Confirmed that with my controller (0x25).